Core/GDCore/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -68,6 +68,10 @@ gd::String EventsCodeGenerator::GenerateRelationalOperatorCall(
     }
   }
 
+  if (relationalOperator == "startsWith") {
+    return rhs + " != '' && " + callStartString + "(" + argumentsStr + ").startsWith(" + rhs + ")";

```

Comment:
`'some text'.startsWith('')` return `true`.
Do you think it's better to return false?

Reply:
I think it's better that this condition `'some text'.startsWith('')` to return false, because I thought it would be easier to understand for non-programmers or for beginners.

In most cases a String Variable is initialized as empty string (in GDevelop can't be initialized as _NULL_), and would be confusing for those people.